decoders/connector/tektelic/seal/v1.0.0/payload.js

```diff
@@ -0,0 +1,1757 @@
+function parsePayload(e) {
```

Comment:
Refactor this file to TypeScript.

Reply:
I must keep it in JS since their decoder is very large and passes the 64kb parser file size when converted to typescript.